<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AKA" failures="1" tests="4" name="com.qa.example.tests.ProductPageTests" time="1.310" errors="0" timestamp="2023-12-12T10:45:48 CST" skipped="0">
  <testcase classname="com.qa.example.tests.ProductPageTests" name="productCountTest" time="0.235"/>
  <system-out/>
  <testcase classname="com.qa.example.tests.ProductPageTests" name="productInfoTest" time="1.036"/>
  <system-out/>
  <testcase classname="com.qa.example.tests.ProductPageTests" name="productPageTitleTest" time="0.017"/>
  <system-out/>
  <testcase classname="com.qa.example.tests.ProductPageTests" name="productInventoryListTest" time="0.022">
    <failure message="lists don&#039;t have the same size expected [6] but found [0]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: lists don't have the same size expected [6] but found [0]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:1017)
at org.testng.Assert.assertEquals(Assert.java:993)
at com.qa.example.tests.ProductPageTests.productInventoryListTest(ProductPageTests.java:43)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- productInventoryListTest -->
  <system-out/>
</testsuite> <!-- com.qa.example.tests.ProductPageTests -->
